tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -426,7 +426,7 @@ public function testUpdatePayment() {
       'financial_trxn_id' => $payment['id'],
     );
     $eft = $this->callAPISuccess('EntityFinancialTrxn', 'get', $params);
-    $amounts = array(66.67, 33.33);
+    $amounts = array(-33.33, -16.67);
```

Comment:
I am not confident why its should be in minus when a payment is updated to 100 from 50!